src/main/java/com/sap/sgs/phosphor/fosstars/model/RatingRepository.java

```diff
@@ -20,6 +23,10 @@
  */
 public class RatingRepository {
 
+  private interface RatingFactory {
+    Rating create() throws IOException;
+  }
+
   /**
    * A logger.
    */
```

Comment:
Can we replace MAPPER.enableDefaultTyping() (Depracated) to  MAPPER.activateDefaultTyping(MAPPER.getPolymorphicTypeValidator());?

Reply:
Yeah, that's a good point. I agree. Yesterday I created #37 to address that. I would prefer to fix it separately.

Reply:
I am not sure it `MAPPER.activateDefaultTyping(MAPPER.getPolymorphicTypeValidator());` is enough. Maybe we'll need to configure the validator more carefully.

Reply:
👍